ipatests/test_ipaserver/test_install/test_bindinstance.py

```diff
@@ -78,19 +78,3 @@ def test_add_crypto_policy(m_set, m_get, namedconf):
     named_add_crypto_policy()
     m_get.assert_called_with('named.conf', 'add_crypto_policy')
     m_set.assert_not_called()
-
-
-@patch('ipaserver.install.sysupgrade.get_upgrade_state')
-@patch('ipaserver.install.sysupgrade.set_upgrade_state')
-def test_add_crypto_policy_no_ipa(m_set, m_get, namedconf):
-    # Test if the update step is skipped when named.conf doesn't contain
-    # IPA related settings.
-    m_get.return_value = False
-    with open(namedconf, 'w') as f:
-        f.write(TEST_CONFIG)
-
-    result = named_add_crypto_policy()
-    assert not result
-
-    m_get.assert_not_called()
-    m_set.assert_not_called()
```

Comment:
Why did you remove this test case?

Reply:
I moved this change out to its own commit.

Reply:
`test_add_crypto_policy_no_ipa` checks if migration was skipped if named.conf was not IPA-configured, after refactoring this check is performed for all the `named_*` migration functions.

Reply:
Thanks that makes sense!